.travis.yml

```diff
@@ -4,14 +4,15 @@ language: node_js
 matrix:
   include:
 
-    # Run linter once
+    # Run linter only, just once
     - node_js: '6'
       env: LINT=true
 
     # Run tests
-    - node_js: '4'
-    - node_js: '5'
+    - node_js: 'node'
```

Comment:
also because then it'd be easy to forget to add `7` back in. Either way you have to update this  when 8 is released